cypress/support/pages/home.pageObject.js

```diff
@@ -3,9 +3,21 @@ import PageObject from '../PageObject';
 class HomePageObject extends PageObject {
   url = '/#/';
 
+  visitYourFeed() {
+    cy.visit('/#/my-feed/');
+  }
+  
   get usernameLink() {
     return cy.getByDataCy('username-link');
   }
+
+  get navBar() {
+    return cy.getByDataCy('header-navbar');
+  }
+
+  assertDeletingArticle(modalText) {
+    cy.get('.article-preview').should('contain', modalText);
```

Comment:
The assertDeletingArticle method seems to have business logic which is not recommended in a PageObject. PageObjects should only provide elements and not perform assertions or checks.